docs/decisions/0003-name-affirmation-signals.rst

```diff
@@ -0,0 +1,38 @@
+3. Using Signals for Name Affirmation
+-------------------------------------
+
+Status
+------
+
+Accepted
+
+Context
+-------
+
+A `VerifiedName` object has a `status` field that should correspond to the status of the associated IDV or proctored exam attempt.
+The value for the field should therefore be affected by any updates to the status of either the IDV or proctored exam attempt. It
+should also be noted that there is not a direct correlation between the statuses for a `VerifiedName` and the statuses
+used by the IDV and proctoring services, so some level of translation has to occur between the services.
+
+Decision
+--------
+
+* Use Django Plugin Signals to send and receive info about IDV and proctored exam attempts for name affirmation to consume.
+
+  * The signals that each service sends should be defined within that service.
+  * Signal receivers will be defined in the name affirmation service.
```

Comment:
Nit: Name affirmation isn't a true service, so we should describe it as a plugin, more accurately.